app/workers/merge_worker.rb

```diff
@@ -5,18 +5,42 @@ class MergeWorker
   include Redisable
   include DatabaseHelper
 
-  def perform(from_account_id, into_account_id)
+  def perform(from_account_id, into_id, type = 'home')
```

Comment:
Why use a string for `type` here? In ruby a symbol would be customary and also save a tiny bit of memory / performance overhead.

Much less important, but it might be nice to use keyword arguments for `type`.

Reply:
This has to do with this being a Sidekiq worker, and Sidekiq serializing the parameters in JSON. Worker parameters are basically hashes and strings and cannot be anything else. It also does not support keyword arguments, and will pass a hash instead.

Reply:
I see. Thanks for clarifying!